internal/rootcoord/root_coord.go

```diff
@@ -1016,8 +1027,7 @@ func (c *Core) Register() error {
 		}
 	})
 
-	c.UpdateStateCode(internalpb.StateCode_Healthy)
-	log.Debug("RootCoord start successfully ", zap.String("State Code", internalpb.StateCode_Healthy.String()))
+	log.Info("RootCoord start successfully ", zap.String("State Code", internalpb.StateCode_Healthy.String()))
```

Comment:
here and in other places: State Code --> state code

Reply:
Ok, I will fix the above comments